.changeset/wise-candles-exercise.md

```diff
@@ -0,0 +1,19 @@
+---
+"@salt-ds/data-grid": patch
+"@salt-ds/lab": patch
+---
+
+RadioButton
```

Comment:
no longer extends HTMLAttributes<HTMLDivElement>, should that be noted clearly?

Reply:
Not sure. Haven't seen any changesets mentioning this. @joshwooding any thoughts on this?

Reply:
I haven't checked but does the move from div to label break anything?